test/test_Benchmark-Theta.jl

```diff
@@ -0,0 +1,164 @@
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PL†X")]) ≈ 56.77852579
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PK†X")]) ≈ 53.42328265
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PL†Y")]) ≈ 18.59487971
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PK†Y")]) ≈ 31.49287695
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PX†U")]) ≈ 96.30435869        #atol=1.0e-4
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PY†U")]) ≈ 53.91109057
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PX‡X")]) ≈ 100
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PY‡Y")]) ≈ 50
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PU‡U")]) ≈ 150
+        @test MPSGE.Complementarity.result_value(m._jump_model[:RA]) ≈ 150
+        @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PUρRA")]) ≈ 140.5066 atol=1.0e-4
+
+        set_value(endow, 1.1)
+        set_fixed!(RA, true)
+        solve!(m)
+        
+                @test value(m, :X) ≈ 0.95023091
+                @test MPSGE.Complementarity.result_value(m._jump_model[:Y]) ≈ 1.04922969
+                @test MPSGE.Complementarity.result_value(m._jump_model[:U]) ≈ 0.98214655
+                @test MPSGE.Complementarity.result_value(m._jump_model[:PX]) ≈ 1.10148665
+                @test MPSGE.Complementarity.result_value(m._jump_model[:PY]) ≈ 0.99755723
+                @test MPSGE.Complementarity.result_value(m._jump_model[:PU]) ≈ 1.06569297
+                @test MPSGE.Complementarity.result_value(m._jump_model[:PL]) ≈ 1.01330185
+                @test MPSGE.Complementarity.result_value(m._jump_model[:PK]) ≈ 0.98719697
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PL†X")]) ≈ 59.29239153
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PK†X")]) ≈ 50.71690507
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PL†Y")]) ≈ 19.68924135
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PK†Y")]) ≈ 30.31483881
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PX†U")]) ≈ 96.75042055                #atol=1.0e-4
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PY†U")]) ≈ 53.41512898
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PX‡X")]) ≈ 100
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PY‡Y")]) ≈ 50
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PU‡U")]) ≈ 150
+                @test MPSGE.Complementarity.result_value(m._jump_model[:RA]) ≈ 157
+                @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PUρRA")]) ≈ 147.322 atol=1.0e-4
+        
+    end
```

Comment:
The entire file is not properly formatted, all the indenting seems inconsistent? Makes it hard to read.

Reply:
oops